Agendas/23_09_22.md

```diff
@@ -0,0 +1,78 @@
+# TEAM MEETING 
+
+## Schedule 
+1. **15:30** Find chairs, clean EME 1252. (Few keen people)
+2. **16:00** Liaisons meet client at fountain statue in courtyard. Rest of the group assembles in EME 1252
+3. **16:10** Commence Meeting. All PMs and Liaisons should be present. 
+
+## OVERVIEW
+
+**CONTEXT:** Kukarella text to speech converter gives you easy access to 750 AI voices across 130 languages and accents. Kukarella users can create a professional voiceover in real-time with multiple languages. Imagine that you are listening to a blog post read by an AI voice. You hear the voice, but you also want to see the person who is reading this text, sort of a personal assistant. Today, there are few solutions which can be used to enable this. You can use pre-recorded videos and sync them with the text; or you can use AI to animate photographs of people or hand-drawn images. We would prefer the latter one because it gives more freedom to a user.
+
+**DESCRIPTION:** The team will build a web app compatible with Kukarella’s tech stack. The output of the web app should be something that is embeddable into a website, ideally an image or video. The input should ideally be via a web HTTP request or as a JS API. Kukarella's code base is written in TypeScript, with a React front-end and a NodeJS Express backend API server; as long as the input is accessible from our backend in some way, it is acceptable. Ideally, for a user, the process of creating a speaking portrait would look something like this: The user enters the text he wants to listen to; uploads a photo of a person or illustration of a character who will read the text; selects a language and voice; presses the "speak" button; and then he can watch and listen to the resulting animated image with a voice over. Tech for illustrations (Links to an external site.), It's source code (Links to an external site.), Tech for photosLinks to an external site. used in my heritage.com
+
+## Personal Questions 
+- Tell us about you and your company (goals and values)
+- Have you been a part of a capstone project or anything similar before?
+- What is your preferred method of communication?
+- How often would you like us to communicate with you outside of our meetings?
+- Are there other people on your team that you’d like us to work with?
+- What is the purpose of this project?
+- What expectations do you have in regard to working with us?
+
```

Comment:
Please add:
- Are there any privacy requirements or concerns for this project? (i.e., do we need to sign a contract, does the repo need to be private?)